### DISCUSSION TITLE
Need help hiding syntax from the render

### DISCUSSION THREAD

**Discussion by probabilityzero** [mProjectsCode/obsidian-meta-bind-plugin#441] [Q&A] [answered]
Hello there, 

I can't seem to figure why the syntax is visible at the bottom, I try to find everything related to it. It's still the same even in reading mode. 

![Screenshot_2024-10-16-12-08-30-89_51606159b24eff83e24a54116878fe3e.jpg](https://github.com/user-attachments/assets/b43ea3ad-001d-42b0-8329-2a8f311b90d5)

**Comment by mProjectsCode** [comment, marked as answer]
remove the `showcase` argument
